database/postgres/postgres_league.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+from database.models import League
+from database.postgres.postgres_country import save_country
+
+def save_league(league_data, session, sport_id) -> League :
```

Comment:
model/class here as well

and also don't forget about datatypes (e.g. league_data: dict)